server/background_tasks.go

```diff
@@ -73,12 +73,16 @@ func (s *Server) ListenForIntegrationsUpdateEvents() error {
 				s.Errorf("ListenForIntegrationsUpdateEvents: " + err.Error())
 				return
 			}
-			systemKeysCollection.UpdateOne(context.TODO(), bson.M{"key": "ui_url"},
-				bson.M{"$set": bson.M{"value": integrationUpdate.UIUrl}})
-			UI_url = integrationUpdate.UIUrl
 			switch strings.ToLower(integrationUpdate.Name) {
 			case "slack":
-				notifications.CacheSlackDetails(integrationUpdate.Keys, integrationUpdate.Properties)
+				systemKeysCollection.UpdateOne(context.TODO(), bson.M{"key": "ui_url"},
+					bson.M{"$set": bson.M{"value": integrationUpdate.UIUrl}})
+				UI_url = integrationUpdate.UIUrl
+				CacheDetails("slack", integrationUpdate.Keys, integrationUpdate.Properties)
+				s.Warnf("save cache slack details %s %s", integrationUpdate.Keys, integrationUpdate.Properties)
+			case "s3":
+				CacheDetails("s3", integrationUpdate.Keys, integrationUpdate.Properties)
+				s.Warnf("save cache s3 details %s %s", integrationUpdate.Keys, integrationUpdate.Properties)
```

Comment:
delete it, I meant to warn in the default block because in this case it means we've got an unknown integration name